build/Settings.cs

```diff
@@ -19,11 +19,11 @@ private static string config(string @default = null, [CallerMemberName] string k
                 : value;
         }
 
-        public const string DotnetIsolatedItemTemplatesVersion = "4.0.2945";
-        public const string DotnetIsolatedProjectTemplatesVersion = "4.0.2945";
-        public const string DotnetItemTemplatesVersion = "4.0.2945";
-        public const string DotnetProjectTemplatesVersion = "4.0.2945";
-        public const string TemplateJsonVersion = "3.1.1648";
+        public const string DotnetIsolatedItemTemplatesVersion = "4.0.3038";
+        public const string DotnetIsolatedProjectTemplatesVersion = "4.0.3038";
+        public const string DotnetItemTemplatesVersion = "4.0.3038";
+        public const string DotnetProjectTemplatesVersion = "4.0.3038";
+        public const string TemplateJsonVersion = "4.0.3038";
```

Comment:
You don't have to update the TemplateJsonVersion.

Reply:
Fixed